flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/utils/CatalogTableStatisticsConverter.java

```diff
@@ -61,6 +63,27 @@ public static TableStats convertToTableStats(
         return new TableStats(rowCount, columnStatsMap);
     }
 
+    public static TableStats convertToAccumulatedTableStates(
+            List<CatalogTableStatistics> tableStatisticsList,
+            List<CatalogColumnStatistics> catalogColumnStatisticsList) {
+        final Optional<TableStats> rowCountMergedTableStats =
+                tableStatisticsList.stream()
+                        .map(p -> CatalogTableStatisticsConverter.convertToTableStats(p, null))
+                        .reduce((s1, s2) -> s1.merge(s2));
+
+        final Optional<TableStats> columnStatsMergedTableStats =
+                catalogColumnStatisticsList.stream()
+                        .map(
+                                p ->
+                                        CatalogTableStatisticsConverter.convertToTableStats(
+                                                CatalogTableStatistics.EMPTY, p))
+                        .reduce((s1, s2) -> s1.merge(s2));
```

Comment:
We can just convert a CatalogTableStatistics and its corresponding  CatalogColumnStatistics into a TableStats, and then call TableStats#merge. If that, CatalogTableStatistics.EMPTY, StatisticDataUtils can be removed. There's no calculation here, and even CatalogColumnStatisticDataBuilder is no needed. This pr will become very simple